Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager
 ###############################################################################
 # Stage 2: Copy build assets to create the smallest final runtime image
 ###############################################################################
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4 AS runtime
+FROM scratch AS runtime
```

Comment:
I think the most suitable UBI image equivalent to `scratch` here would be `ubi-micro`, i.e. `registry.access.redhat.com/ubi8/ubi-micro:8.7` but I have not used it before. 

We should make sure that a minimal set of tools/CLIs are available to log on to a shell on the pod, `cat` a config or log file, etc.

What do you think @njhill